features/orders.feature

```diff
@@ -0,0 +1,41 @@
+Feature: List orders
+  As Giga-Food
+  In order to make a decision to start an order
+  I like to have three options approve, cancel and a input filed with transport size
+
+  Background:
+    Given an admin exists with email "admin@example.com" and password "password"
+    And I'm loged in as admin user "admin@example.com"
+    Given I go to the dashboard
```

Comment:
Change to And?
In the test there are several inconsistent step. Check again.

"Given" for background steps
"When" for actions 
"Then" for assertions
Use "And" for repetting the same type of step

Reply:
Done